### ISSUE TITLE
Option to hide 'Implement missing function' option

### ISSUE THREAD

**Issue by Tasin5541** [microsoft/vscode-copilot-release#3381]
Any plans to add an option to remove this option from quick fix?

![Image](https://github.com/user-attachments/assets/b1160ffc-16dc-4c87-b43b-c1b4c9452860)

I have set `Enable Code Actions` to false, but that doesn't seem to hide this option.

**Comment by blbqli** [comment]
I thought, that's crazy, too!

**Comment by Tasin5541** [comment]
> This is from the TypeScript extension: https://github.com/microsoft/vscode/blob/613ad56601c62ff8fea04a62444217470fbc496d/extensions/typescript-language-features/src/languageFeatures/quickFix.ts#L380-L381

Do we need to open a thread there?

**Comment by mjbvz** [comment]
Yes, please open an issue there. Also make sure to explain why you want to disable it
